spec/support/factory_girl.rb

```diff
@@ -7,7 +7,7 @@
     password_confirmation 'password'
     first_name "John"
     last_name "Doe"
-    sequence(:phone_number) { |n| "555#{n}#{n}#{n}#{n}#{n}22" }
+    sequence(:phone_number) { |n| "5558273862" }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.